src/command_helpers.rs

```diff
@@ -130,6 +130,8 @@ impl StderrForwarder {
                             self.bytes_available_failed = true;
                             MIN_BUFFER_CAPACITY
                         }
+                        #[cfg(target_family = "wasm")]
+                        Err(_) => unimplemented!(),
```

Comment:
Is this matching necessary?

Can't we return it directly?

Reply:
Hey sorry could you clarify what "it" is in this case? If I remember correctly, I think the main issue here is that on wasm platforms, the match statement ends up missing an `Err` case and won't compile as a result. To be honest, I wasn't too sure about the logic behind this match statement since it seems to conditionally return a `usize` or a `bool` depending on the platform and whether the parallel feature is enabled. But my naive assessment was that some sort of panic would make sense here for the error case when targeting wasm 😅

Reply:
I think we should not call this function on wasm.

The `bytes_available` function should also be put behind a `cfg(not(target_family = "wasm"))`

Reply:
Took another look and it seems ok to panic, because the `bytes_available` always return `Ok()`.

```suggestion
                        #[cfg(target_family = "wasm")]
                        Err(_) => panic!("bytes_available should always succeed on wasm"),
```